ntbs-integration-tests/NotificationPages/MDRDetailsPageTests.cs

```diff
@@ -218,22 +218,40 @@ public async Task ValidateMDRDetailsProperty_ReturnsErrorIfDescriptionContainsNu
         }
 
         [Theory]
-        [InlineData("10000", "The NTBS ID does not match an existing ID in the system")]
-        [InlineData("1e1", "The NTBS ID must be an integer")]
-        public async Task ValidateMDRDetailsRelatedNotification_ReturnsErrorIfInvalidId(string attemptedId, string errorMessage)
+        [InlineData(Utilities.DRAFT_ID)]
+        [InlineData(Utilities.DENOTIFIED_ID)]
+        [InlineData(Utilities.NEW_ID)]

```

Comment:
We shouldn't be able to link to draft/denotified cases either, so shouldn't just check whether id does not exist but also the state